AnkiDroid/build.gradle

```
@@ -224,6 +224,14 @@ tasks.withType(JavaCompile) {
     options.incremental = true
 }
 
+// Install Git pre-commit hook for Ktlint
+task installGitHook(type: Copy) {
+    from new File(rootProject.rootDir, 'pre-commit')
+    into { new File(rootProject.rootDir, '.git/hooks') }
+    fileMode 0777
```

Should not be world/group writable? 

```suggestion
    fileMode 0755
```